app/views/style_guides/_tags.haml

```diff
@@ -40,3 +40,10 @@
     = tagged(true, 'warning', title: 'Status', label: 'Parked', icon: 'clock-o')
   .d-flex.mr-3
     = tagged(true, 'danger', title: 'Status', label: 'Pre-empted', icon: 'cross')
+.mt-8.d-flex.flex-column
+  - (1..14).each do |id|
+    - next if id == 5
+    .mt-3.ml-6
+      .enrollment__project_type{class: "client__service_type_#{id}"}
+        .service-type__program-type{data: {toggle: :tooltip, title: HUD.project_type(id)}}
+          = HUD.project_type_brief(id)
```

Comment:
Here's a before and after on the project types. All of them are passing WCAG AAA contrast requirements now.

# before
<img width="204" alt="Screen Shot 2022-06-23 at 5 56 28 PM" src="https://user-images.githubusercontent.com/5333982/175342733-e2672779-7781-45dd-8c74-edc807ae4200.png">


# after

<img width="389" alt="Screen Shot 2022-06-23 at 5 50 55 PM" src="https://user-images.githubusercontent.com/5333982/175342358-103661fa-d80c-4556-a66c-67fc7891b469.png">